book/website/reproducible-research/testing/testing-exceptions.md

```diff
@@ -182,4 +182,5 @@ Unit test frameworks for other languages also often provide similar functions:
 - JUnit for Java: org.junit.Assert.assertEquals(double expected, double actual, double delta)
 - testthat for R:
   - expect_equal(actual, expected, tolerance=DELTA) - absolute error within DELTA
-  - expect_equal(actual, expected, scale=expected, tolerance=DELTA) - relative error within DE L T A
+  - expect_equal(actual, expected, scale=expected, tolerance=DELTA) - relative error within DELTA
+- julia: `isapprox` or `≈`
```

Comment:
I suggest adding them as list and showing the parameters, like
- `val1 ≈ val2`
- `isapprox(val1, val2, atol=abs_delta, rtol=rel_delta)`
- Test.jl with `≈`: `@test val1 ≈ val2 atol=abs_delta rtol=rel_delta`